devito/cgen_utils.py

```diff
@@ -158,8 +158,8 @@ def _print_ListInitializer(self, expr):
     def _print_IntDiv(self, expr):
         return expr.__str__()
 
-    def _print_Byref(self, expr):
-        return "&%s" % expr.name
+    _print_Byref = _print_IntDiv
```

Comment:
really? Looks odd to me

Reply:
yes because ``__str__()`` is exactly what we get to generate